packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -632,7 +637,6 @@ export default class TradeStore extends BaseStore {
 
         if (!ObjectUtils.isEmptyObject(requests)) {
             this.proposal_requests = requests;
-            this.proposal_info     = {};
             this.purchase_info     = {};
 
             Object.keys(this.proposal_requests).forEach((type) => {
```

Comment:
shouldn't we update the req id of the proposal request we are sending here so we can check in response?

Reply:
the `requests` obj here doesn't have req_id unfortunately, it comes in the response